src/main/java/net/sf/jabref/exporter/SavePreferences.java

```diff
@@ -51,7 +51,7 @@ public static SavePreferences loadForExportFromPreferences(JabRefPreferences pre
     }
 
     public static SavePreferences loadForSaveFromPreferences(JabRefPreferences preferences) {
-        Boolean saveInOriginalOrder = true;
```

Comment:
I'm not really sure I'm commenting on the right thing, but I am sure that the consensus is that the default should be to save in original order. It might be that this switch cannot be changed in the preferences, but then that issue should be solved.